drivers/sensor/stm32/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Kconfig - STM32F401x temperature sensor configuration options
+
+#
+# Copyright (c) 2016 ARM Ltd.
+# Copyright (c) 2017 Linaro Limited.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig TEMP_STM32F401X
+	bool
+	prompt "STM32F401x Temperature Sensor"
+	depends on SENSOR && SOC_FAMILY_STM32
```

Comment:
Driver seems to be specific to STM32F4, should it depend on it instead of FAMILY_STM32?